docs/book/v2/intro.md

```diff
@@ -0,0 +1,30 @@
+# Introduction
+
+This library provides the ability to dispatch middleware pipelines in place of
+controllers within [laminas-mvc](https://docs.laminas.dev/laminas-mvc/).
+
+## Dispatching PSR-7 Middleware
+
+[PSR-7](http://www.php-fig.org/psr/psr-7/) defines interfaces for HTTP messages,
+and is now being adopted by many frameworks; Laminas itself offers a
+parallel microframework targeting PSR-7 with [Mezzio](https://docs.mezzio.dev/mezzio).
+What if you want to dispatch PSR-7 middleware from laminas-mvc?
+
+laminas-mvc currently uses [laminas-http](https://docs.laminas.dev/laminas-http/)
+for its HTTP transport layer, and the objects it defines are not compatible with
+PSR-7, meaning the basic MVC layer does not and cannot make use of PSR-7
+currently.
```

Comment:
At this stage, I'd argue we should be talking less about PSR-7, and more about PSR-15, because that's what v2 of this library is specifically targetting. PSR-7 is simply providing the HTTP messages _consumed by_ PSR-15 implementations.